infra/Pos/DHT/Workers.hs

```diff
@@ -56,6 +56,8 @@ dumpKademliaStateWorker
     => KademliaDHTInstance
     -> (WorkerSpec m, OutSpecs)
 dumpKademliaStateWorker kademliaInst = localOnNewSlotWorker onsp $ \slotId ->
+    -- TODO: [CSL-198] Catch and report non-critical errors, as
+    -- 'onNewSlot' no longer swallows all exception and rethrows them instead.
```

Comment:
I don't see why this worker may fail in normal cases, so let's stop program if it does (i. e. not catch anything here).